node-aci/build

```diff
@@ -0,0 +1,39 @@
+#! /bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+set -o xtrace
+
+rm -f node.aci
+
+docker2aci docker://debian:jessie
```

Comment:
@vishh One reason I avoid Alpine: it lacks security metadata http://lists.alpinelinux.org/alpine-devel/5228.html and https://github.com/coreos/clair/issues/12